packages/core/src/utils/collect-updates/collect-updates.ts

```diff
@@ -29,10 +29,19 @@ export function collectUpdates(filteredPackages: Package[], packageGraph: Packag
       : new Map(filteredPackages.map(({ name }) => [name, packageGraph.get(name)]));
 
   let committish = commandOptions.since;
+  const tagPattern = isIndependent ? '*@*' : '';
+
+  if (hasTags(execOpts, tagPattern)) {
+    const describeOptions: DescribeRefOptions = {
+      ...execOpts,
+    };
+
+    if (isIndependent) {
+      describeOptions.match = tagPattern;
+    }
 
-  if (hasTags(execOpts)) {
     // describe the last annotated tag in the current branch
-    const { sha, refCount, lastTagName } = describeRefSync(execOpts, commandOptions.includeMergedTags, gitDryRun);
+    const { sha, refCount, lastTagName } = describeRefSync(describeOptions, commandOptions.includeMergedTags, gitDryRun);
```

Comment:
I don't have access to the project with work laptop but what are the differences with `execOpts`? Are you adding something to the object or is it completely different object?